fineract-provider/src/main/java/org/apache/fineract/interoperation/service/InteropServiceImpl.java

```diff
@@ -171,17 +171,17 @@ public InteropServiceImpl(PlatformSecurityContext securityContext, InteropDataVa
     private static final class KycMapper implements RowMapper<InteropKycData> {
 
         public String schema() {
-            return " country.code_value as nationality, c.`date_of_birth` as dateOfBirth, c.`mobile_no` as contactPhone, gender.code_value as gender, c.`email_address` as email, "
-                    + "kyc.code_value as idType, ci.`document_key` as idNo, ci.`description` as description, "
+            return " country.code_value as nationality, c.date_of_birth as dateOfBirth, c.mobile_no as contactPhone, gender.code_value as gender, c.email_address as email, "
+                    + "kyc.code_value as idType, ci.document_key as idNo, ci.`description` as description, "
```

Comment:
@galovics any reason why these backticks are still here? Shouldn't they be removed?

Reply:
@vidakovic they will be removed in an upcoming PR, not forgotten but good job spotting it ;-)